packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -68,9 +68,21 @@ export default function EntitiesSavedStates( { close } ) {
 	} = useDispatch( coreStore );
 
 	// To group entities by type.
-	const partitionedSavables = Object.values(
-		groupBy( dirtyEntityRecords, 'name' )
-	);
+	const partitionedSavables = groupBy( dirtyEntityRecords, 'name' );
+
+	// Sort entity groups.
+	const {
+		site: siteSavables,
+		wp_template: templateSavables,
+		wp_template_part: templatePartSavables,
```

Comment:
Do you think that order can be defined as a "priority" somehow in the entities config? Or maybe this is sufficient for now?

Reply:
I think it'd make the code quite a bit more complex, and I don't think it's worth it right now, since this is the only place so far where we're needing that priority order.